README.md

```diff
@@ -23,6 +23,19 @@ This dataset was prepared by Pierre-Luc Carrier and Aaron Courville, as part of
 
 Past Kaggle competition for this partical challenge: https://www.kaggle.com/c/challenges-in-representation-learning-facial-expression-recognition-challenge
 
+##To install
+--On Linux
```

Comment:
just a comment, i want to rewrite this, as soon as i get dlib running - i think andreas is having the same problems on mac btw.. did you get it working?